pkg/ui/workspaces/cluster-ui/src/components/tableMetadataLastUpdated/tableMetadataJobControl.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import { RedoOutlined } from "@ant-design/icons";
+import { Skeleton, Tooltip } from "antd";
+import React, { useCallback, useEffect } from "react";
+
+import {
+  TableMetadataJobStatus,
+  triggerUpdateTableMetaJobApi,
+  useTableMetaUpdateJob,
+} from "src/api/databases/tableMetaUpdateJobApi";
+import Button from "src/sharedFromCloud/button";
+import { usePrevious } from "src/util/hooks";
+
+import styles from "./tableMetadataJobControl.module.scss";
+
+type TableMetadataJobControlProps = {
+  // Callback for when the job has updated the metadata, i.e. the
+  // lastUpdatedTime has changed.
+  onDataUpdated?: () => void;
+};
+
+export const TableMetadataJobControl: React.FC<
+  TableMetadataJobControlProps
+> = ({ onDataUpdated }) => {
+  const { jobStatus, refreshJobStatus, isLoading } = useTableMetaUpdateJob();
+  const previousUpdateCompleted = usePrevious(
+    jobStatus?.lastCompletedTime?.unix(),
+  );
+  const lastUpdateCompletedUnixSecs = jobStatus?.lastCompletedTime?.unix();
+  const lastUpdatedText = jobStatus?.lastCompletedTime?.format(
+    "YYYY-MM-DD HH:mm:ss (UTC)" ?? "Never",
```

Comment:
we don't have a global time format for everything?

also, customers can have a selected timezone that's not UTC

Reply:
Ah that's right - using `FormatWithTimezone` now. Thanks!